### PR TITLE
Add cocoapod flags to another command

### PR DESCRIPTION
**Pull request by crazytonyli**
Following up https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/86#issuecomment-2043868041

---

- [x] I have considered if this change warrants release notes and have added them to the appropriate section in the `CHANGELOG.md` if necessary.

### CODE REVIEW

File: bin/publish_private_pod [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+
 PATCH_COCOAPODS="false"
+COCOAPODS_FLAGS=(--verbose)
 
 while [[ "$#" -gt 0 ]]; do
 	case $1 in
-		--patch-cocoapods) PATCH_COCOAPODS="true" ;;
+		--patch-cocoapods)
+			PATCH_COCOAPODS="true"
+                        ;;
```
Comment (AliSoftware) [inline bin/publish_private_pod]: Indentation inconsistency 🙃

File: CHANGELOG.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ _None._
 
 ### New Features
 
-_None._
+- Make `publish_private_pod` support `--allow-warnings`. [#86]
```
Comment (AliSoftware) [inline CHANGELOG.md]: 💭 Worth introducing Danger to this repo to remind us not to forget to add those entries next time? 🤔

**Comment by crazytonyli** [comment]
Ops, sorry, I didn't noticed this PR contains old commits that are squash-merged into the trunk branch.

**Comment by AliSoftware** [comment]
Even after your force-push to fix the old squash-merged commits, there's still a tab-vs-spaces indentation inconsistency with [line 18 using spaces](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/blob/add-cocoapod-flags-to-another-command/bin/publish_private_pod#L18C15-L18C25) while the rest of the file uses tabs, which might not be super visible on GitHub web/desktop, but makes the `;;` misaligned if one's tab-width is not set to 8 🙃  (I only noticed it because I initially reviewed the PR on GitHub mobile 😅 )

**Comment by crazytonyli** [comment]
🙈 I didn't see that at all. Addressed in https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/87/commits/af60ecda466a2458d01f2410eaef257e97f78ae4

**Review by AliSoftware — APPROVED** [review]

**Pull request merged**